### PR TITLE
adding note and link to federatedIdpMfaBehavior

### PR DESCRIPTION
**Pull request by Justinha**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[1fb4762](https://github.com/Justinha/azure-docs-powershell-azuread/commits/1fb4762ba95e4b6be0a37b49906dce8fda7ac7ca)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-1.0/MSOnline/Confirm-MsolDomain.md](https://github.com/Justinha/azure-docs-powershell-azuread/blob/supports-mfa/azureadps-1.0/MSOnline/Confirm-MsolDomain.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/confirm-msoldomain?view=azureadps-1.0&branch=pr-en-us-803) (azureadps-1.0) |
[azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md](https://github.com/Justinha/azure-docs-powershell-azuread/blob/supports-mfa/azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/set-msoldomainauthentication?view=azureadps-1.0&branch=pr-en-us-803) (azureadps-1.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C7%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204071655179173-803%5Cworkflow_report.html?sv=2020-08-04&se=2022-05-08T16%3A56%3A38Z&sr=b&sp=r&sig=LIbzLUzSmcGuDv4XWN3%2BXfEk4h4qxNLUIWDNYPec3VM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[7dea593](https://github.com/Justinha/azure-docs-powershell-azuread/commits/7dea59341751d45c1bd14b369b07936e69098546)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-1.0/MSOnline/Confirm-MsolDomain.md](https://github.com/Justinha/azure-docs-powershell-azuread/blob/supports-mfa/azureadps-1.0/MSOnline/Confirm-MsolDomain.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/confirm-msoldomain?view=azureadps-1.0&branch=pr-en-us-803) (azureadps-1.0) |
[azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md](https://github.com/Justinha/azure-docs-powershell-azuread/blob/supports-mfa/azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/set-msoldomainauthentication?view=azureadps-1.0&branch=pr-en-us-803) (azureadps-1.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C7%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204071807193828-803%5Cworkflow_report.html?sv=2020-08-04&se=2022-05-08T18%3A11%3A30Z&sr=b&sp=r&sig=TO8mr8UwXsreFA4qIqhXAw0NWMAUI0%2FpYapAO%2BhuwOc%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: azureadps-1.0/MSOnline/Confirm-MsolDomain.md [hunk 1]
```diff
@@ -203,6 +203,10 @@ Accept wildcard characters: False
 ### -SupportsMfa
 Indicates whether the IDP STS supports MFA.
 
+>[!NOTE]
+>We recommend configuring the security setting [federatedIdpMfaBehavior](/graph/api/resources/federatedIdpMfaBehavior?view=graph-rest-beta&preserve-view=true) using Microsoft Graph PowerShell for greater control over how to protect federated domains.
```
Comment (msewaweru) [inline azureadps-1.0/MSOnline/Confirm-MsolDomain.md:207]: Hi @Justinha, not sure this link returns a 404 on staging. Please check.

File: azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md [hunk 2]
```diff
@@ -206,6 +206,10 @@ Accept wildcard characters: False
 ### -SupportsMfa
 Indicates whether the IDP STS supports MFA.
 
+>[!NOTE]
+>We recommend configuring the security setting [federatedIdpMfaBehavior](/graph/api/resources/federatedIdpMfaBehavior?view=graph-rest-beta&preserve-view=true) to prevent bypassing of Azure MFA.
```
Comment (msewaweru) [inline azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md:210]: As per my previous comment @Justinha.

**Review by msewaweru — CHANGES_REQUESTED** [review]
Hi @Justinha, thanks for your contribution. Please check the comments I have left.

**Comment by billmath** [comment]
Docs Build status updates of commit _[7f30474](https://github.com/Justinha/azure-docs-powershell-azuread/commits/7f304742815432da0093fa456c9d49350a9d1d9c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-1.0/MSOnline/Confirm-MsolDomain.md](https://github.com/Justinha/azure-docs-powershell-azuread/blob/supports-mfa/azureadps-1.0/MSOnline/Confirm-MsolDomain.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/confirm-msoldomain?view=azureadps-1.0&branch=pr-en-us-803) (azureadps-1.0) |
[azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md](https://github.com/Justinha/azure-docs-powershell-azuread/blob/supports-mfa/azureadps-1.0/MSOnline/Set-MsolDomainAuthentication.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/msonline/set-msoldomainauthentication?view=azureadps-1.0&branch=pr-en-us-803) (azureadps-1.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C5%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202205051605490823-803%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-05T16%3A06%3A48Z&sr=b&sp=r&sig=wx0bHHZqY%2B5wxHOxCqNPr%2FbWiEyGzFsthfKItF2QP9A%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by Justinha** [comment]
@msewaweru sorry I am trying to link to https://docs.microsoft.com/en-us/graph/api/resources/internaldomainfederation?view=graph-rest-beta#federatedidpmfabehavior-values 

I submitted a new commit but still get a 404. Can you see any syntax error or do cross-repo links result in 404?

**Comment by msewaweru** [comment]
> @msewaweru sorry I am trying to link to https://docs.microsoft.com/en-us/graph/api/resources/internaldomainfederation?view=graph-rest-beta#federatedidpmfabehavior-values
> 
> I submitted a new commit but still get a 404. Can you see any syntax error or do cross-repo links result in 404?

Thanks @Justinha for the commit. I have tested this change and it now works. Once I remove the review. in staging the intended page loads. Thanks.

**Comment by msewaweru** [comment]
@CelesteDG, this is good to go.

**Pull request merged**